### PR TITLE
fix: the relation column name in UserMarketingCta

### PR DESCRIPTION
**Pull request by omBratteng**
Didn't realise that typeorm had added a extra column for foreign key

### CODE REVIEW

File: src/entity/user/UserMarketingCta.ts [hunk 1]
```diff
@@ -17,9 +17,10 @@ export class UserMarketingCta {
   readAt: Date;
 
   @ManyToOne(() => MarketingCta, {
-    lazy: true,
+    eager: true,
```
Comment (capJavert) [inline src/entity/user/UserMarketingCta.ts:20]: Why do we need this? It means relation will be loaded each time we query the table through `find`. Which should not be the case.

**Review by capJavert — CHANGES_REQUESTED** [review]
Pesky typeorm 😄 Good thing we added epic branch 👌

File: src/entity/user/UserMarketingCta.ts (hunk 1 above)
Comment (omBratteng) [reply src/entity/user/UserMarketingCta.ts:20]: Wasn't supposed to add eager to it

Comment (omBratteng) [reply src/entity/user/UserMarketingCta.ts:20]: Intended to use `relations: ['marketingCta'],` in the findOne but was testing to see what was the difference and forgot to revert it

**Review by capJavert — APPROVED** [review]

**Pull request merged**